packages/analytics/src/public-types.ts

```diff
@@ -288,4 +288,33 @@ export interface EventParams {
   page_path?: string;
   [key: string]: unknown;
 }
+
+/**
+ * Consent status settings for each consent type.
+ * For more information, see
+ * {@link https://developers.google.com/tag-platform/tag-manager/templates/consent-apis
+ * | the GA4 reference documentation for consent state and consent types}.
+ * @public
+ */
+export interface ConsentSettings {
+  // Enables storage, such as cookies, related to advertising
+  ad_storage?: ConsentStatusString;
+  // Enables storage, such as cookies, related to analytics (for example, visit duration)
+  analytics_storage?: ConsentStatusString;
+  // Enables storage that supports the functionality of the website or app such as language settings
```

Comment:
Looking at the generated analytics.api.md, it looks like api-documenter still thinks these are undocumented. I think api-documenter may need comments to be in the /** */ format.